Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian:bullseye-slim
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends curl jq ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl https://fp.dev/fp/v0.8.0/x86_64-unknown-linux-gnu/fp -o /usr/bin/fp
```

Comment:
nitpick, maybe we can move the version to a dockerfile ARG, that makes it easier to update down the line: https://docs.docker.com/engine/reference/builder/#arg

Reply:
Oh that is very cool. The problem is we cannot pass build arguments to Docker as GitHub actions builds it automatically.